### PR TITLE
Emoji Classification using OpenCV

### PR DESCRIPTION
**Pull request by abhisheks008**
## Related Issue #49 
The project is developed to create a Machine Learning model which will predict the emotion of an user given image and also classify the emotion such as Happy, Sad, Angry and many more. Also the project has to be done using OpenCV and additionally I have used the Tensorflow and Keras for creating the model more accurate!

- [x] LGM-SOC'21 Participant 
- [x] Contributor

Closes: #49 

### Description
The project is developed to create a Machine Learning model which will predict the emotion of an user given image and also classify the emotion such as Happy, Sad, Angry and many more. Also the project has to be done using OpenCV and additionally I have used the Tensorflow and Keras for creating the model more accurate!

## Type of change

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
The accuracy of the model is checked using the accuracy score and the cross validation score.
|Accuracy Metric|Score|
|:---:|:---:|
|Accuracy Score|71.73|
|CV Score|65.22|

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.

- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by abhisheks008** [comment]
@prathimacode-hub I have deployed the model of Emoji Classification using OpenCV. Please check it.

Thank you!

**Review by prathimacode-hub — APPROVED** [review]
Perfect. You had done a fab work. Good choice of the project. And very well executed. 🏷👌✔@abhisheks008
